bint.go

```diff
@@ -300,14 +292,21 @@ func parseLargeToU128(s []byte) (u128, uint8, error) {
 		return coef, 0, nil
 	}
 
+	// now 0 < pos < l-1
+	// nolint: gosec
```

Comment:
```suggestion
	//nolint:gosec
```

Reply:
Is it a standard? What's the difference between this and having space?

Reply:
I replied in the previous PR

Reply:
Thanks. Seems like a solid reason to me. I'll change other `nolint` places as well since they're all having spaces

Reply:
Adding the nolintlint linter to your golangci-lint conf would help you to detect them

Reply:
yep, will add it also